src/comgr.cpp

```diff
@@ -305,10 +303,15 @@ static void RemoveLinkOptionsUnwanted(OptionList& list)
 } // namespace hip
 
 /// \todo Get list of supported isa names from comgr and select.
-static std::string GetIsaName(const std::string& device)
+static std::string GetIsaName(const miopen::TargetProperties& target)
 {
-    const char* const ecc_suffix = IsEnabledFeatureSramEcc(device) ? "+sram-ecc" : "";
-    return {"amdgcn-amd-amdhsa--" + device + ecc_suffix};
+#if ROCM_FEATURE_TARGETID_OFF
+    const char* const ecc_suffix = target.Sramecc() ? "+sram-ecc" : "";
```

Comment:
Shouldn't it be this instead?
```suggestion
    const char* const ecc_suffix = target.Sramecc() ? ":sramecc+" : "";
```

Reply:
On further thought, I see this is in the "ROCM_FEATURE_TARGETID_OFF" section of the code, so I think my change request is incorrect.  The code here is pre-targetid, and it was working there.  Right?

Reply:
Yes!